WalletWasabi.Fluent/ViewModels/HelpAndSupport/AboutViewModel.cs

```diff
@@ -17,73 +17,79 @@ namespace WalletWasabi.Fluent.ViewModels.HelpAndSupport;
 	Category = "Help & Support",
 	Keywords = new[]
 	{
-			"About", "Software", "Version", "Source", "Code", "Github", "Status", "Stats", "Tor", "Onion", "Bug",
-			"Report", "FAQ", "Questions,", "Docs", "Documentation", "Link", "Links", "Help"
+		"About", "Software", "Version", "Source", "Code", "Github", "Status", "Stats", "Tor", "Onion", "Bug",
+		"Report", "FAQ", "Questions,", "Docs", "Documentation", "Link", "Links", "Help"
 	},
 	NavBarPosition = NavBarPosition.None,
 	NavigationTarget = NavigationTarget.DialogScreen)]
 public partial class AboutViewModel : RoutableViewModel
 {
-	public AboutViewModel()
+	public AboutViewModel() : this(false)
 	{
+	}
+
+	public AboutViewModel(bool navigateBack)
+	{
+		EnableBack = navigateBack;
```

Comment:
```suggestion
		EnableBack = Navigate().CanNavigateBack;
```

if `Navigate().CanNavigateBack;` works as expected.

Reply:
sadly it didnt worked :(